src/lib_json/json_writer.cpp

```diff
@@ -15,79 +15,12 @@
 #include <sstream>
 #include <utility>
 
```

Comment:
remove this blank line and merge cmath and cstdio into the previous block of include headers.